pkg/services/apiserver/auth/authorizer/provider.go

```diff
@@ -20,6 +21,7 @@ type GrafanaAuthorizer struct {
 
 func NewGrafanaAuthorizer(cfg *setting.Cfg, orgService orgsvc.Service) *GrafanaAuthorizer {
 	authorizers := []authorizer.Authorizer{
+		authorizerfactory.NewPrivilegedGroups("system:masters"),
```

Comment:
any concerns if we always allow requests from the loopback's `system:masters` group?